### ISSUE TITLE
Devottools install Segmentation error

### ISSUE THREAD

**Issue by yr542** [davidaknowles/leafcutter#241]
I am trying to install leafcutter using the recommended command:

```
devtools::install_github("davidaknowles/leafcutter/leafcutter”)
```

However it gives me the following interesting error:

```
Segmentation fault
* removing path/to/leafcutter
Warning message:
In i.p(...)  :
  installation of package path/to/leafcutter had non-zero exit status
```

What do I do to get around this issue?

**Comment by mjhemmerlein** [comment]
I installed rstan and rstantools through install.packages.  I then tried to install using devtools::install_github("davidaknowles/leafcutter/leafcutter") and received the same error.

**Comment by TheWizier** [comment]
Been trying to install this in a docker container for a while now, and I found this thread:
https://www.reddit.com/r/bioinformatics/comments/1601hfr/installation_of_leafcutter_fails_with_fatal_error/
"It's because the tbb/task_scheduler_init.h is deprecated and was removed from the latest release of TBB, that's why the error message is about not finding the header file"

**Comment by yr542** [comment]
Yes but the issue remains that because of the error it cannot be installed.

**Comment by yaqisu** [comment]
Hi I wonder have you solved the problem? I also got the same issue and still cant figure it out...

**Comment by yaqisu** [comment]
> Hi I wonder have you solved the problem? I also got the same issue and still cant figure it out...

nvm I followed the instruction [here](https://github.com/davidaknowles/leafcutter/issues/188) and it finally works!
